etna/pipeline/pipeline.py

```diff
@@ -56,7 +56,7 @@ def _forecast(self) -> TSDataset:
 
         if isinstance(self.model, DeepBaseModel):
             future = self.ts.make_future(
-                future_steps=self.model.decoder_length, transforms=self.transforms, tail_steps=self.model.encoder_length
+                future_steps=self.horizon, transforms=self.transforms, tail_steps=self.model.encoder_length
```

Comment:
Why do you fix it here? May be it is better of merge this fix from master during merging.

Reply:
I have recently rebased on master branch, this is a small artefact